src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDatabaseMetaData.java

```diff
@@ -257,12 +257,18 @@ private void checkClosed() throws SQLServerException {
     private static final String IS_GENERATEDCOLUMN = "IS_GENERATEDCOLUMN";
     private static final String IS_AUTOINCREMENT = "IS_AUTOINCREMENT";
     private static final String ACTIVITY_ID = " ActivityId: ";
+    
+    private static final String NVARCHAR = "NVARCHAR";
+    private static final String VARCHAR = "VARCHAR";
+    private static final String INTEGER = "INT";
+    private static final String SMALLINT = "SMALLINT";
```

Comment:
instead of adding these extra string defns, why not just use the actual sql types?